mods/hide-dotfiles-explorer.wh.cpp

```diff
@@ -0,0 +1,690 @@
+// ==WindhawkMod==
+// @id              hide-dotfiles-explorer
+// @name            Hide Dotfiles (Explorer only) v1.0.1
```

Comment:
There's no need to add the version to the name, the `@version` field is used for that.